terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperViewTest.java

```diff
@@ -0,0 +1,207 @@
+
+public class ReservationReportPdfStamperViewTest {
+
+    private static final SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
+
+    private static final int OUTPUT_BYTE_ARRAY_INITIAL_SIZE = 4096;
+
+    ReservationReportPdfStamperView reservationReportPdfStamperView;
+
+    Map<String, Object> model;
+
+    PdfStamper stamper;
+
+    HttpServletRequest request;
+
+    HttpServletResponse response;
+
+    @Before
+    public void setUp() {
+        reservationReportPdfStamperView = new ReservationReportPdfStamperView();
+
+        DownloadPDFOutput downloadPDFOutput = new DownloadPDFOutput();
+        downloadPDFOutput.setReferenceName("TERASOLUNA TRAVEL CUSTOMER CENTER");
+        downloadPDFOutput.setReferenceEmail("customer@example.com");
+        downloadPDFOutput.setReferenceTel("01-2345-6789");
+        downloadPDFOutput.setPaymentMethod("Bank Transfer");
+        downloadPDFOutput.setPaymentAccount("Current Account：12345678");
+        downloadPDFOutput.setChildCount(3);
+        downloadPDFOutput.setTourName("【おすすめ】Terasolunaツアー(6泊7日)");
+        downloadPDFOutput.setAccomName("TERASOLUNAホテル第一荘");
+        downloadPDFOutput.setCustomerKana("キムラ　タロウ");
+        downloadPDFOutput.setCustomerTel("111-1111-1111");
+        downloadPDFOutput.setAdultUnitPrice(75000);
+        downloadPDFOutput.setConductor("Yes");
+        downloadPDFOutput.setTourAbs(
+                "そこは別天地、静寂と湯けむりの待つ宿へ… 詳しい情報はお取り合わせをお願い致します。");
+        downloadPDFOutput.setCustomerAdd("千葉県八千代市上高野");
+        downloadPDFOutput.setCustomerJob("プログラマ");
+        downloadPDFOutput.setTourDays("7");
+        downloadPDFOutput.setCustomerName("木村　太郎");
+        downloadPDFOutput.setChildUnitPrice(37500);
+        downloadPDFOutput.setDepName("北海道");
+        downloadPDFOutput.setArrName("北海道");
+        downloadPDFOutput.setCustomerMail("tarou@example.com");
+        downloadPDFOutput.setAdultCount(5);
+        downloadPDFOutput.setCustomerCode("00000001");
+        downloadPDFOutput.setReserveNo("00000001");
+        downloadPDFOutput.setRemarks("特になし");
+        downloadPDFOutput.setAccomTel("018-123-4567");
+        downloadPDFOutput.setCustomerPost("276-0022");
+        downloadPDFOutput.setAdultPrice(375000);
+        downloadPDFOutput.setChildPrice(112500);
+        downloadPDFOutput.setSumPrice(487500);
+        downloadPDFOutput.setPaymentTimeLimit("2019/03/24");
+        try {
+            downloadPDFOutput.setReservedDay(sdf.parse("2019/02/21"));
+            downloadPDFOutput.setDepDay(sdf.parse("2019/03/31"));
+            downloadPDFOutput.setCustomerBirth(sdf.parse("1975/01/05"));
+            downloadPDFOutput.setPrintDay(sdf.parse("2019/03/06"));
+        } catch (ParseException e) {
+            e.printStackTrace();
+            fail(); // FAIL when exception is thrown
+        }
+
+        model = new HashMap<String, Object>();
+        model.put("downloadPDFOutput", downloadPDFOutput);
+        model.put("downloadPDFName", "reservationReport");
```

Comment:
- [ ] Why setup `model` at `@Before`? `model` is used only `testMergePdfDocument`.

Reply:
b7c739f6c7a55250b7a1b9951b5ba091b399b99d

I fixed in above commit .